.github/workflows/ppa_automation.yml

```diff
@@ -0,0 +1,68 @@
+name: PPA Automated Releases
+on:
+  push:
+    branches:
+      - 'releases/**'
```

Comment:
also the `main` one?

Reply:
I'd like to see automated builds happening off `main` as well, but they could be a little more risky and we need to make sure they go to a different PPA from the release candidates. Otherwise we could get ourselves into trouble while if the release branches have a lower version number than main.

For example, we are releasing version 2.3.3 imminently while `main` is on 2.4.0